File: back/back/Models/Game.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace back.Models
{
    public class Game
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int Players { get; set; }
        public int YearOfProduction { get; set; }
        public string Description {get;set;}
        public string Type {get;set;}

    }
}

File: front/src/app/models/Game.ts
export class Game{
    id:number;
    name:string;
    players:number;
    yearOfProduction:number;
    description:string;
    type:string;

    constructor( name, players, year, desc, type)
    {
        this.name = name;
        this.players = players;
        this.yearOfProduction = year;
        this.description = desc;
        this.type;
    }

}